athenahealth/enum/appointment_status.go

```diff
@@ -0,0 +1,41 @@
+package enum
+
+import "strconv"
+
+// AppointmentStatus is derived from https://docs.athenahealth.com/api/api-ref/appointment#Book-appointment
+type AppointmentStatus string
+
+const (
+	Cancelled     AppointmentStatus = "x"
+	ChargeEntered AppointmentStatus = "4"
+	CheckedIn     AppointmentStatus = "2"
+	CheckedOut    AppointmentStatus = "3"
+	Future        AppointmentStatus = "f"
+	Open          AppointmentStatus = "o"
+)
+
+var appointmentStatusToString = map[AppointmentStatus]string{
+	Cancelled:     "x",
+	ChargeEntered: "4",
+	CheckedIn:     "2",
+	CheckedOut:    "3",
+	Future:        "f",
+	Open:          "o",
+}
+
+func (as AppointmentStatus) Int() int {
+	intVal, convErr := strconv.Atoi(appointmentStatusToString[as])
+	if convErr != nil {
+		return -1
+	}
+
+	return intVal
+}
+
+func (as AppointmentStatus) String() string {
+	return appointmentStatusToString[as]
```

Comment:
I think we can do away with `appointmentStatusToString` in favor of `string(as)`

Reply:
I'm not sure how we tell if a regular string is a valid AppointmentStatus or not but I'll make this change